apps/browser/src/key-management/biometrics/foreground-browser-biometrics.ts

```diff
@@ -34,7 +36,7 @@ export class ForegroundBrowserBiometricsService extends BiometricsService {
     if (!response.result) {
       return null;
     }
-    return response.result;
+    return new SymmetricCryptoKey(Utils.fromB64ToArray(response.result.keyB64)) as UserKey;
```

Comment:
how about this one ?
```suggestion
    return new SymmetricCryptoKey.fromString(response.result.keyB64) as UserKey;
```